documentation/sphinx/source/mr-status-json-schemas.rst.inc

```diff
@@ -633,7 +633,7 @@
             },
             "memory":{
                "free_bytes":0, // an estimate of how many bytes are free to allocate to fdbservers without swapping
-               "committed_bytes":0,
+               "committed_bytes":0, // an estimate of the number of bytes of memory on the machine that are not available
```

Comment:
I'm confused at the comment.
Do you mean `an estimate of how many in-use bytes in memory that are not available on the machine`? or
`the machine is not available, and this provides the estimation of the memory size in bytes`?

I think you meant the former one, but it causes some confusion to me in the first place. Maybe it is just because my English is not that good.

Reply:
I meant the former in this case, it was motivated by AJ's clarification here: https://forums.foundationdb.org/t/few-queries-on-status-json/684/2

Reply:
I guess in status, what I called "available" is actually called "free". @xumengpanda Would it be clearer if we did the following:

```suggestion
               "committed_bytes":0, // an estimate of the number of bytes of memory on the machine that are not free for allocation
```